app/helpers/application_helper.rb

```diff
@@ -4,4 +4,16 @@ def categories
     Category.all
   end
 
+  def item_status_check(item)
+    if item.status == "active"
+      button_to "Add to Cart", carts_path(item_id: item.id)
+    else
+      button_to "Item Retired", retired_carts_path(item_id: item.id)
+    end
+  end
+
+  def format_price_per_unit(item)
```

Comment:
Thoughts on whether this should go in the helper too. Do we use this in the controller or just the view? @wlffann